app/src/components/main/modal/modalLayout/modalFooter/modalFooter.scss

```diff
@@ -40,9 +40,12 @@
   }
 }
 .buttons-block {
-  float: right;
   white-space: nowrap;
+  display: flex;
+  justify-content: space-between;
```

Comment:
This code affects all modals, please revert it and refer to my commentary from above